src/action/linux/selinux/nix.te

```diff
@@ -8,4 +8,9 @@ require {
   type etc_t;
   type var_run_t;
   type systemd_unit_file_t;
-}
\ No newline at end of file
+  type default_t;
+  type init_t;
+  class lnk_file read;
+}
+
+allow init_t default_t:lnk_file read;
```

Comment:
I'm surely no expert with SELinux, so this might be an obvious question but ... did this policy _do_ anything before? Why did we have it?